hack/test_k8s_e2e.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#check whether KUBECONFIG variable is set or not
+set -e
+# set environment variable and Kubeconfig
+KUBECONFIG=${KUBECONFIG:-""}
+LOCATION=${LOCATION:-""}
+
+if [ -z "$LOCATION" ]
+then
+echo "Location is not set"
+exit 1
+fi
+
+if [ -z "$KUBECONFIG" ]
+then
+export KUBECONFIG=$GOPATH/src/k8s.io/cloud-provider-azure/_output/$(ls -t _output | head -n 1)/kubeconfig/kubeconfig.$LOCATION.json
+#export Kubeconfig path
+fi
+
+ [ ! -d "$GOPATH/src/k8s.io/kubernetes" ] && cd $GOPATH/src/k8s.io && git clone https://github.com/kubernetes/kubernetes.git
```

Comment:
replace `cd $GOPATH/src/k8s.io && git clone https://github.com/kubernetes/kubernetes.git` with

```
git clone https://github.com/kubernetes/kubernetes.git $GOPATH/src/k8s.io/kubernetes
```

In this way, if `$GOPATH/src/k8s.io` doesn't exist, it would be created automatically

Reply:
okay sir but after that we need to add
`cd $GOPATH/src/k8s.io/kubernetes`

Reply:
right, cd is still required

Reply:
Done Sir